CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Changelog
 
+### v1.5.16
+- Updating Falcon Crowdstrike to 5.29.
```

Comment:
@gkallenberg I see v1.5.16 updates the footer. Do I include in the same version or add a new version? Thanks!

Reply:
Add a new patch version with your changes -- bump to v1.5.17